autoload/utils/exec/job.vim

```diff
@@ -97,18 +88,26 @@ function! s:nVimExit(job_id, data, event) abort
         return
     endif
     call s:createQuickFix()
+    call s:closeBuffer()
     if a:data != 0
         copen
     endif
 endfunction
 
 function! s:createJobBuf() abort
-    silent execute 'belowright 10split ' . s:cmake4vim_buf
+    let l:in_quickfix = getwininfo(win_getid())[0]['quickfix']
```

Comment:
It the cursor is in a `quickfixlist`, use the same window for the output instead of moving it to the bottom (nothing happens if it's already at the bottom, but it doesn't move it there if the `qflist` is to the right or left, for example).